src/Core/src/MauiApp.cs

```diff
@@ -5,7 +5,7 @@ namespace Microsoft.Maui
 {
 	public abstract class MauiApp : App
 	{
-		public abstract IWindow GetWindowFor(IActivationState state);
+		public abstract IWindow CreateWindowFor(IActivationState state);
```

Comment:
If you think about it, this is not a very .NET name. Why can't we just use `CreateWindow`?

Reply:
Changed